src/main/java/org/example/constants/DatabaseConstants.java

```diff
@@ -0,0 +1,7 @@
+package org.example.constants;
+
+public class DatabaseConstants {
+  public static final String URL = "jdbc:mysql://127.0.0.1:3306/item_data";
+  public static final String USER = "root";
```

Comment:
can you check other way to store username and password rather than plaintext. Check how can we store such details in application.yml in spring boot

Reply:
Added these details to application.yml file